kroxylicious-operator/src/main/resources/META-INF/fabric8/kafkaservices.kroxylicious.io-v1.yml

```diff
@@ -37,13 +37,39 @@ spec:
               type: object
             spec:
               type: object
-              required: ["bootstrapServers"]
+              required: [ "bootstrapServers" ]
               properties:
                 bootstrapServers:
                   description: Bootstrap servers of the Kafka cluster.
                   type: string
                   minLength: 1
                   pattern: ^(([^:]+:[0-9]{1,5}),)*([^:]+:[0-9]{1,5})$
+                nodeIdRanges:
```

Comment:
I wonder if we can get any goodness out of CEL, like if we could validate the uniqueness of the range names. We should probably also validate the ranges don't overlap somewhere, maybe in the

Reply:
after investigation it appears CEL is not a good fit as comparing all array items to each other is too expensive. Lets add this to the KafkaServiceReconciler in a subsequent PR.